src/main/java/github/scarsz/discordsrv/objects/managers/link/JdbcAccountLinkManager.java

```diff
@@ -79,16 +80,22 @@ public static boolean shouldUseJdbc(boolean quiet) {
         if (StringUtils.isBlank(jdbc)) return false;
 
         Matcher matcher = JDBC_PATTERN.matcher(jdbc);
-        if (!matcher.find() || matcher.groupCount() < 4) {
-            if (!quiet) DiscordSRV.error("Not using JDBC because < 4 matches for JDBC url");
+        
+        if (!matcher.matches()) {
+            if (!quiet) DiscordSRV.error("connection string '" + jdbc + "' is NOT valid!");
+            return false;
+        }
+        
+        if (!matcher.group("DBEngine").equalsIgnoreCase("jdbc")) {
+            if (!quiet) DiscordSRV.error("Not using JDBC because the protocol of the JDBC URL is wrong!");
             return false;
         }
 
         try {
-            String engine = matcher.group(1);
-            String host = matcher.group(2);
-            String port = matcher.group(3);
-            String database = matcher.group(4);
+            String engine = matcher.group("DBEngine");
+            String host = matcher.group("DBHost");
+            String port = matcher.group("DBPort");
+            String database = matcher.group("DBName");
```

Comment:
- `DB` before the group names is unnecessary
- Lowercase these

Reply:
done